src/Tribe/Settings_Manager.php

```diff
@@ -323,7 +323,7 @@ public function add_help_admin_menu_item() {
 			return;
 		}
 
-		$parent = class_exists( 'Tribe__Events__Main' ) ? Tribe__Settings::$parent_page : Tribe__Settings::$parent_slug;
+		$parent = class_exists( 'Tribe__Events__Main' ) && Tribe__Events__Main::POSTTYPE;
```

Comment:
This turns a string into a boolean

Reply:
@Camwyn I've realised that we need to deprecate the method since it was replaced by `Settings::add_admin_pages()` in TEC